ydb/public/tools/lib/cmds/ut/test.py

```diff
@@ -24,3 +24,17 @@ def test_kikimr_config_generator_generic_connector_config():
 
     actual = generic_connector_config()
     assert actual == expected
+
+def test_merge_two_yaml_configs():
```

Comment:
It needs to test differnt cases with unit tests: dicts, lists, dict in list, list in dict, combinations with None, etc.

Canionization only one of tests